src/main/resources/public/js/app.js

```diff
@@ -3,19 +3,22 @@ var gameModel;
 //This function will be called once the page is loaded.  It will get a new game model from the back end, and display it.
 $( document ).ready(function() {
 
-  $.getJSON("model", function( json ) {
-    displayGameState(json);
-    gameModel = json;
+    var audio = new Audio('audio/Music.mp3');
+    audio.play();
+
+  $.getjson("model", function( json ) {
```

Comment:
-I'm not sure what did those changes, I think my IDE did that automatically because I never touched those. I'll change that